### PR TITLE
freshen the k8s service tutorial so it's compatible with any k8s

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: docusaurus/docs/learn/quickstarts/services/kubernetes-service.md [hunk 1]
```diff
@@ -2,69 +2,81 @@
...
-   Now you have two services available to your tunneler:
-   * hello-service
-   * testapi-service
+    ```text
+    helm install "hello-toy" openziti/hello-toy \
+        --namespace hello-toy --create-namespace \
+        --set serviceDomainName=hello
+    ```
 
+1. Visit the Hello Demo page in your browser: [http://hello.ziti.internal/](http://hello.ziti.internal/)
```
Comment (dovholuknf) [inline docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:82]: you skipped a step to install an identity?

Comment (qrkourier) [reply docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:82]: It's on line 27 (attempted diff link: https://github.com/openziti/ziti-doc/pull/873/files#diff-3ff9ef69e1e2c17b10e58b3eff919a73ad8b3ef5beb557cc0a3b39e6f7ddae17R27)

Comment (dovholuknf) [reply docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:82]: nope - i missed it when reviewing. i see it up there now

File: docusaurus/docs/learn/quickstarts/services/kubernetes-service.md [hunk 2]
```diff
@@ -2,69 +2,81 @@
...
-        --service-roles '@hello-service' --identity-roles '#hello-hosts'
+    ```text
+    ziti edge create identity "hello-client" \
+        --role-attributes hello-clients \
+        --jwt-output-file hello-client.jwt
+    ```
 
-    ziti edge create service-policy "hello-dial-policy" Dial \
-        --service-roles '@hello-service' --identity-roles '#hello-clients'
+1. Ensure you have router policies for your identities and services.
```
Comment (dovholuknf) [inline docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:35]: i feel like this step is "just noise". i can understand why you put it here but i'd probably remove it. that or maybe you could move this step to the end and instead, just use `policy-advisor identities -q` ?

File: docusaurus/docs/learn/quickstarts/services/kubernetes-service.md [hunk 3]
```diff
@@ -2,69 +2,81 @@
...
-        --jwt-output-file /tmp/hello-host.jwt --role-attributes hello-hosts
+    ziti edge list edge-routers 'isTunnelerEnabled=true'
+    ```
 
-    ziti edge enroll /tmp/hello-host.jwt
+    If none of the routers shown are in your cluster, then refer to [the router deployment guide](/guides/deployments/30-kubernetes/kubernetes-router.mdx) for more information on how to deploy a router.
 
-    ziti edge create config "hello-intercept-config" intercept.v1 \
-        '{"protocols":["tcp"],"addresses":["minihello.ziti"], "portRanges":[{"low":80, "high":80}]}'
+1. Add a role to the router's tunneler identity. This labels the identity so we can give it permission to host the service as a reverse-proxy. Substitute the router name for `router1`.
```
Comment (dovholuknf) [inline docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:20]: I'd change the wording just a bit here. I'd start by saying "after selecting the router from the output of the above command, add a role to the router's identity allowing it to host the demo service. For example, if the router found was named `router1`, the command to issue is shown below

something liek that that just whacks them  over the head a bit more

**Review by dovholuknf — APPROVED** [review]

File: docusaurus/docs/learn/quickstarts/services/kubernetes-service.md (hunk 3 above)
Comment (qrkourier) [reply docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:20]: https://github.com/openziti/ziti-doc/pull/874

File: docusaurus/docs/learn/quickstarts/services/kubernetes-service.md (hunk 2 above)
Comment (qrkourier) [reply docusaurus/docs/learn/quickstarts/services/kubernetes-service.md:35]: https://github.com/openziti/ziti-doc/pull/874

**Pull request merged**